lib/Twig/Extensions/SimpleTokenParser.php

```diff
@@ -8,6 +8,12 @@
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  */
+
+@trigger_error('The grammar feature is deprecated since version 1.5 and will be removed in 2.0.', E_USER_DEPRECATED);
```

Comment:
the message need some context: in the stream of all deprecation notices of your apps, you may miss where this comes from